README.md

```diff
@@ -103,3 +103,11 @@ To enable pre-commit in your development environment:
     pre-commit install --hook-type pre-commit
     pre-commit install --hook-type commit-msg
     ```
+## Documentation
+HTML documentation can be generated by typing:
+

```

Comment:
We should list the dependencies here: 
- obviously we need to have Doxygen
- but is it required to have other tools like Graphviz ? (I had some issue when generating the doc).